BEIMA.Backend/ReportService/ReportWriter.cs

```diff
@@ -350,7 +355,6 @@ private static List<string> GenerateDeviceReportHeaders(DeviceType deviceType)
         /// </summary>
         /// <param name="device">Device that's data values should be used</param>
         /// <param name="deviceType">DeviceType that the field keys should be based on</param>
-        /// <param name="buildings">Buildings that the building id should be based on</param>
```

Comment:
Why remove this?

Reply:
Not sure how that got removed. Will get it back in.